module01/ex05/ex05.md

```diff
@@ -1,37 +1,84 @@
-# Exercise 05 - Working with lists
+# Exercise 05 - Bank Account
 
-|                         |                                 |
-| -----------------------:| ------------------------------- |
-|   Turn-in directory:    |  ex05                           |
-|   Files to turn in:     |  eval.py                        |
-|   Forbidden functions   |  while                          |
-|   Remarks:              |  use zip & enumerate            |
+|                         |                     |
+| -----------------------:| ------------------- |
+|   Turn-in directory:    |  ex05/              |
+|   Files to turn in:     |  the_bank.py        |
+|   Forbidden functions:  |  None               |
+|   Remarks:              |  n/a                |
 
 ## Objective:
-The goal of the exercise is to discover 2 useful methods for listsm tuplesm dictionnaries (iterable class
-objects more generally) named `zip` and `enumerate`.
+The goals of this excercise is to discover new built-in functions, deepen the class manipulation and
+to be aware of the posibility to modify istanced objects.
```

Comment:
posibility -> possibility
istanced -> instanced

Reply:
not in the subject for holygraph